systemtests/bin/setup_download_tests.py

```diff
@@ -199,75 +221,79 @@ def setup_download_tests(start_page, auth_token, csv_output_path, zip_output_dir
     them to two separate zip folders: one for try symbols files, and one for
     regular symbols files. This is used for the download system tests.
 
+    Runtime is dependent on what files have been uploaded recently.
+
     Note: This requires an auth token for symbols.mozilla.org to view files.
 
     """
     # We want at least one SYM file from the /try bucket and one from each platform,
     # so initialize a map to keep track of what file types we have
     sym_file_type_to_filename = {}
 
-    click.echo("Fetching symbols files ...")
-    # Get a try symbol first. Since these are more rare, add a key filter in the URL
-    sym_files_url = urljoin(SYMBOLS_URL, "/api/uploads/files?key=try/")
-    end_condition_num_files = 1
-    iterate_through_symbols_files(
-        auth_token,
-        sym_files_url,
-        start_page,
-        sym_file_type_to_filename,
-        end_condition_num_files,
-    )
+    with METRICS.timer("elapsed_time"):
```

Comment:
This whole section is wrapped in a timer which spits out the elapsed time after the wrapped block is done. Thus the next bunch of lines are all indented.